content/PIV/Guides/Android_code_signing.adoc

```diff
@@ -44,6 +60,9 @@ Certificate fingerprint (SHA1): 26:D7:CB:71:6D:42:3C:AB:58:69:E0:9D:F0:16:DF:84:
 ....
 +
 Password here is the PIN of the key (default 123456).
++
+If the keytool command fails, adding the -debug option might help understand the problem. One common issue is that the slots might be indexed differently (sarting from 0, for example), requiring the file created on the step above to be configured in another way.
```

Comment:
"starting" rather than "sarting"

maybe "created in the step..." rather than "created on the step..."